### ISSUE TITLE
Add function PoE 

### ISSUE THREAD

**Issue by wybaux** [PowerAruba/PowerArubaSW#71]
Hi, 

Do you think it's possible to add in the next release a way to get the statut of the PoE on a port (enable or disable) ? 

Thank you, 

Nicolas Wybaux

**Comment by alagoutte** [comment]
Hi Nicolas,

Sure, it will be possible... what do you need ? only Poe Status ? or also the power consumption ?

**Comment by wybaux** [comment]
Hi Alexis,
Yes only the PoE Status should be perfect.
A big thank for your great job :)

**Comment by alagoutte** [comment]
if you want only the status, you can already use the following cmdlet : 

```powershell
((Invoke-ArubaSWWebRequest -method "get" -url "rest/v4/poe/ports").content | ConvertFrom-Json).port_poe
```

**Comment by wybaux** [comment]
Thank you, it's working as expected.

**Comment by alagoutte** [comment]
> 
> 
> Thank you, it's working as expected.

Ok, it will be add "later"

**Comment by alagoutte** [comment]
Hi Nicolas,

PoE cmdlet (Get-ArubaSWPoE/Set-ArubaSWPoE/Get-ArubaSWPoEStats) will be available on new release (0.8) (planned for next weeks...)
